process/accountProcessor.go

```diff
@@ -327,6 +327,37 @@ func (ap *AccountProcessor) GetKeyValuePairs(address string, options common.Acco
 	return nil, ErrSendingRequest
 }
 
+// GetGuardianData returns the guardian data for the given address
+func (ap *AccountProcessor) GetGuardianData(address string, options common.AccountQueryOptions) (*data.GenericAPIResponse, error) {
+	observers, err := ap.getObserversForAddress(address)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, observer := range observers {
+		apiResponse := data.GenericAPIResponse{}
+		apiPath := AddressPath + address
```

Comment:
do we need this `AddressPath` to be exported? i think it can be unexported

Reply:
done